src/semantic-ui/site/collections/table.overrides

```diff
@@ -38,6 +38,13 @@ Table Overrides - REACT-INVENIO-APP-ILS
 
 @{fs-parent-selector},
 @{bo-parent-selector} {
+
+  .document-metadata-tabs{
```

Comment:
It seems the `.overrides` files are not formatted in some cases there is a following space after names other times not. A quick fix is to open them as `Less` file type in VsCode, then prettier will do the rest.